examples/xmlns-declarations/xmlns_declarations.bal

```diff
@@ -1,17 +1,25 @@
 import ballerina/io;
 
-// The identifier followed by the `as` keyword is the prefix bound
-// to this namespace name.
+// The identifier followed by the `as` keyword is the prefix bound to the namespace name.
+// Here `eg` is bound to `"http://example.com"`.
 xmlns "http://example.com" as eg;
 
-xmlns "http://example.com" as ex;
-
 public function main() {
+    // The prefixes to which namespaces declarations are bound to can be used in XML templates.
     xml x = xml `<eg:doc>Hello</eg:doc>`;
 
-    boolean b = (x === x.<ex:doc>);
+    // `xmlns` declarations are also allowed at block level.
+    // The `ex` prefix is also bound to `"http://example.com"`.
```

Comment:
```suggestion
    // The `ex` prefix is also bound to `http://example.com`.
```

Reply:
It's kinda implied that it is the string.